src/Symfony/Component/Console/Application.php

```diff
@@ -454,13 +454,15 @@ public function add(Command $command)
     public function get($name)
```

Comment:
has() is affected also, which might simplify get() as such

```php
if (!$this->has($name)) {
   throw ...;
}

$command = $this->commands[$name];
```

Reply:
Calling `has()` should not initialize the command if it exists to me (only `get()` should do), so unless I'm mistaken, making it return false because `!$this->get($name)->isEnabled()` would be wrong

Reply:
well.. for non-lazy commands has() returns false in case it's disabled. I dont see a real reason why that wouldnt apply to lazy commands.

Reply:
I agree, just feel bad to load before `get()` :) 
@ogizanagi agree for `Application::has()` calling `CommandLoaderInterface::get()`?

Reply:
Yes, it looks the right thing to do to me regarding the existing functional.